### ISSUE TITLE
Deployment number/status anomaly for `0x00...00`

### ISSUE THREAD

**Issue by OlivierBBB** [Consensys/besu-sequencer-plugins#1059]
## Ranges

```rust
6871261-6871263
```

## Issue

Faulty range: `6871261-6871263`. Some more details:

```java
// Absolute transaction number & hub stamp

hub.txStack.currentAbsNumber() = 109
hub.stamp() = 291905 // and likely earlier

// Anomaly

Address zeroAddress = Address.fromHexString( "00" )

hub.deploymentNumberOf( zeroAddress ) = 1
hub.deploymentStatusOf( zeroAddress ) = true
```

<img width="695" alt="image" src="https://github.com/user-attachments/assets/c2ded6a7-a0b2-4a74-8885-6696aa541ba4">

**Comment by OlivierBBB** [comment]
Fixed in #1051

**Comment by OlivierBBB** [comment]
Fixed in #1051
